code/modules/security_levels/keycard_authentication.dm

```diff
@@ -169,6 +169,7 @@
 	addtimer(CALLBACK(src, PROC_REF(reset)), confirm_delay)
 
 /obj/machinery/keycard_auth/proc/trigger_event()
+	SHOULD_NOT_SLEEP(TRUE) // exploits related to ERT spawning can come up here related to ERT spawning if this sleeps
```

Comment:
This is barely used throughout the codebase excluding signal handlers, I'm not sure if we should use these codewise when needed or not

Reply:
I think it's fine. Also can you reword this comment a bit, maybe briefly describe the issue?